run/local-run.sh

```diff
@@ -112,6 +112,11 @@ if [[ ! -z "$GATEWAY" &&
   PRERUN_COMMAND="kubectl port-forward service/che-host ${CHE_FORWARDED_PORT}:8080 -n $CHE_NAMESPACE"
 fi
 
+DASHBOARD_POD_NAME=$(kubectl get pods -n eclipse-che -o=custom-columns=:metadata.name | grep che-dashboard)
+export SERVICE_ACCOUNT_TOKEN=$(kubectl exec --stdin --tty $DASHBOARD_POD_NAME -n eclipse-che  -- cat /run/secrets/kubernetes.io/serviceaccount/token)
+export CHECLUSTER_CR_NAMESPACE=eclipse-che
+export CHECLUSTER_CR_NAME=eclipse-che
```

Comment:
should those be configurable via env vars?

Reply:
I have fixed it.